airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -336,33 +335,36 @@ def fail_task(
 
     def dag_started(
         self,
-        dag_run: DagRun,
-        msg: str,
+        dag_id: str,
+        execution_date: datetime,
```

Comment:
This argument (same for other functions) should be called `logical_date` going forward.

Reply:
@uranusjr fixed this in https://github.com/apache/airflow/pull/41889

Reply:
Is it backwards-compatible with Airflow 2? (note that providers will not be usable for Airflow 3 for ~ 6 months and they should be Airflow 2 compatible.

Reply:
Yes - `logical_date` has been a property of DagRun for a long time.

Reply:
Yeah. I know it's there - same as in #41889  - the question is whether it will not change semantics of the produced openlineage events because `execution_date` and `logical_date` are not 1-1. 

So - I am not against it, but the question is whether **some** explanation in changelog should be given to the users when we do the change.